src/Components/App/App.js

```diff
@@ -40,7 +40,7 @@ class App extends Component {
     if (!this.state.isFavorite) {
       this.setState(prevState => ({ 
         ...prevState,
-        savedReadings: [...prevState.savedReadings, {[Date.now()]: prevState.cards}],
+        userSavedReadings: [...prevState.userSavedReadings, {[Date.now()]: prevState.cards, comments: ''}],
```

Comment:
Dope use of spread operator and the prevState here! I don't have much practice with prevState but now I want to try it out, seems very useful!